### ISSUE TITLE
Example of using env from lockbox secrets

### ISSUE THREAD

**Issue by iunir** [turokg/yappa#39]
I'm stuck of configuring yappa to use environment variables from lockbox secrets. 
Is it currently supported? Do you have any config examples?

**Comment by svetlyak40wt** [comment]
I also interested in this feature.

**Comment by turokg** [comment]
nah. sorry guys for late response, can't make github notifications to work properly

as to lockbox or other 'proper secrets handling' - no, there is no support at the moment
